profiles/factory.py

```diff
@@ -42,7 +42,7 @@ def create_app(config: Config, clients: Clients) -> Flask:
 
     config_bus = dict(config.bus)
     config_bus['env'] = config.name
-    publisher = get_publisher(pub_name='profiles', config=config_bus)
+    publisher = get_publisher(config=config_bus)
```

Comment:
Update to the library caused this, do we need to set it elsewhere @seanwiseman?

Reply:
Ah yes. The instance storage and ref by name had to go, was not adding much anyway.

No, it does not need to be added anywhere else.